scheduler.py

```diff
@@ -12,42 +12,47 @@
 
 raven_client = RavenClient()
 
+logger = logging.getLogger(__name__)
 
 # When testing changes, set the "TEST_SCHEDULE" envvar to run more often
 if os.getenv("TEST_SCHEDULE"):
     schedule_kwargs = {"hour": "*", "minute": "*/10"}
 else:
     schedule_kwargs = {"hour": 4}
 
-logging.basicConfig()
```

Comment:
Does this logger still work correctly? IIRC I had to call `basicConfig()` to point it at STDERR and get any output.